db/migrate/20210826080108_create_reporting_patient_follow_ups.rb

```diff
@@ -0,0 +1,7 @@
+class CreateReportingPatientFollowUps < ActiveRecord::Migration[5.2]
+  def change
+    create_view :reporting_patient_follow_ups, materialized: true
+
+    add_index :reporting_patient_follow_ups, [:patient_id, :user_id, :facility_id, :month_date], unique: true, name: "reporting_patient_follow_ups_unique_index"
```

Comment:
This unique index covers all four columns of the table in recommended order of most-values to least-values, necessary for concurrent refresh.

Reply:
👍